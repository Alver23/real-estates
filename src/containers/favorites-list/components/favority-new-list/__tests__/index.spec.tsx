// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import FavorityNewList from '../index';

describe('<FavorityNewList />', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<FavorityNewList />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render correctly', () => {
    expect(component.find('span').exists()).toBeTruthy();
  });
});
