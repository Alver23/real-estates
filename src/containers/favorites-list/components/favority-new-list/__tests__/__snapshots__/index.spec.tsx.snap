// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavorityNewList /> should save a snapshot of the component 1`] = `
<Fragment>
  <Card
    containerStyle=" bg-blue-100 h-full text-blue-400 flex justify-center items-center"
  >
    <VscAdd
      className="text-blue-400"
      size={44}
    />
  </Card>
  <div
    className="flex flex-col justify-center items-center mt-4"
  >
    <span
      className="text-lg text-blue-400 font-medium"
    >
      Crear una nueva lista
    </span>
  </div>
</Fragment>
`;
